samples/nrf9160/mqtt_simple/src/main.c

```diff
@@ -509,6 +510,31 @@ static void button_handler(uint32_t button_states, uint32_t has_changed)
 }
 #endif
 
+static int shell_mqtt_publish(const struct shell *shell, size_t argc, char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	int ret;
+
+	ret = data_publish(&client,
+                        MQTT_QOS_1_AT_LEAST_ONCE,
+                        CONFIG_BUTTON_EVENT_PUBLISH_MSG,
+                        sizeof(CONFIG_BUTTON_EVENT_PUBLISH_MSG) - 1);
```

Comment:
You should use combination of tabs and spaces, not spaces only - see compliance failure:
```suggestion
	ret = data_publish(&client,
			   MQTT_QOS_1_AT_LEAST_ONCE,
			   CONFIG_BUTTON_EVENT_PUBLISH_MSG,
			   sizeof(CONFIG_BUTTON_EVENT_PUBLISH_MSG) - 1);
```